tests/tests.js

```diff
@@ -148,7 +148,7 @@
             assert.expect(9);
             image.src = imageUrl;
 
-            $(image).load(function() {
+            $(image).ready(function() {
```

Comment:
According to [jQuery docs](https://api.jquery.com/ready/), `.ready()` is for document loading. How does it work with image?